bin/link_check.py

```diff
@@ -39,14 +43,15 @@ def test_ftp(url):
         req = urllib.request.Request(url)
         if proxy:
             req.set_proxy(proxy, 'http')
-        response = urlopen(req, timeout=30)
+        response = urlopen(req, timeout=URL_TIMEOUT)
         chunk = response.read(16)
         if len(chunk) == 16:
             res.status_code = 200
         else:
             res.status_code = 404
     except:
-        res.status_code = 404
+        print('ftp exception', url)
+        return Exception()
```

Comment:
Ok, then at least use `except Exeption as e:` to avoid the KeyboardInterrupt one and how about returning the actual exception instead of a bare one in case the caller wants to do more with the actual error?